db/seeds.rb

```diff
@@ -1,7 +1,59 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+pizzerias = Yelp.client.search("Boston", { term: "pizza", 
+                                           category_filter: "pizza",
+                                           sort: 2 })
+
+pizzerias.businesses.each do |pizzeria|
+  pizza = Pizzeria.create(name: pizzeria.name, 
+                   street: pizzeria.location.address[0], 
+                   city: pizzeria.location.city, 
+                   state: pizzeria.location.state_code,
+                   zip_code: pizzeria.location.postal_code,
+                   phone: pizzeria.display_phone,
+                   website: pizzeria.url,
+                   photo: pizzeria.image_url)
+  puts "Pizzeria #{pizza.name} created"
+end
+
+pizzerias = Yelp.client.search("Somerville", { term: "pizza", 
+                                           category_filter: "pizza",
+                                           sort: 2 })
+
+pizzerias.businesses.each do |pizzeria|
+  pizza = Pizzeria.create(name: pizzeria.name, 
+                   street: pizzeria.location.address[0], 
+                   city: pizzeria.location.city, 
+                   state: pizzeria.location.state_code,
+                   zip_code: pizzeria.location.postal_code,
+                   phone: pizzeria.display_phone,
+                   website: pizzeria.url,
+                   photo: pizzeria.image_url)
+  puts "Pizzeria #{pizza.name} created"
+end
+pizzerias = Yelp.client.search("Cambridge", { term: "pizza", 
+                                           category_filter: "pizza",
+                                           sort: 2 })
+pizzerias.businesses.each do |pizzeria|
+  pizza = Pizzeria.create(name: pizzeria.name, 
+                   street: pizzeria.location.address[0], 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.